### ISSUE TITLE
Flying projectiles damage items on the ground far too often.

### ISSUE THREAD

**Issue by arockperson** [CleverRaven/Cataclysm-DDA#45479]
# Describe the bug

Firing full auto guns at creatures standing on a tile with items on the ground damages the items on the ground to a (in my opinion) absurd extent.

# Steps To Reproduce

1. Spawn an item and put it on the ground.
2. Place a zombie on top of the item.
3. Fire a full auto gun at the zombie.

# Expected behavior

Flying bullets shouldn't hit things on the ground nearly as often. It seems insanely unlikely to hit things on the ground below what you are shooting at--even tiny things which are even unlikelier to get hit.  This happens even with amazing skill and stability, which doesn't seem sensible since surely after killing your target the bullets should continue flying instead of hitting the ground.

# Screenshots

<!-- **If applicable**, add screenshots to help explain your problem.
![image](https://user-images.githubusercontent.com/48835780/99455212-8c3f7680-28f5-11eb-9a2e-a3c55f3685d5.png)

This is the result of firing a mere 4 shots with a SCAR-H at a zombie standing directly over a cash card, a cell phone, a cigarette, a candy cigarette, and a cigar. I also did the same thing with a zombie standing on top of a SCAR-H, which almost completely destroyed the gun on the ground in one burst.

**Comment by kevingranade** [comment]
How often do you think it should happen? It happens "too much" doesn't give us anything to go on.

**Comment by arockperson** [comment]
If I had to make a guess, it should happen somewhere from 1/100 to 1/1000 as often. 

But now that I think about it some more, I don't think it's really a problem of the probability of it happening, but more with the overall simulation of projectiles. It seems, correct me if I'm wrong, that if you fire a burst at a zombie and kill it, the remaining shots of the burst will currently act as if you targeted the ground, instead of (logically, I think) continuing to fly past the zombie (or perhaps hitting the corpse before it has time to fall, but that's pretty complicated). I've tested out burst firing with the A7 laser rifle because of the visible beam, and it looks like that's how it currently works--either that or the corpse of the zombie is supposed to absorb the remaining shots, but that doesn't explain the very high damage to items on the ground.


Please excuse the artistic brilliance I am about to post.
<img width="699" alt="trueart" src="https://user-images.githubusercontent.com/48835780/99989463-a8716680-2d80-11eb-9115-f8fe4b115afd.png">

**Comment by kevingranade** [comment]
Good point with the edit, the trajectory is definitely wrong for almost* ever hitting items at the feet of a standing target.

Perhaps we should record a bit more information about the intended trajectory.

* if your dispersion is high enough it becomes a possibility, but a very unlikely one.
